### PR TITLE
release 2.0

### PR DESCRIPTION
**Pull request by appars**
Hi, I am have changed release version from 1.0 to 2.0. Can you please review and approve it?
Thanks
Appar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
-IBM DevOps training... 
 
-
-Release 1.0 on 11th March 2018
+Release 2.0 on 26th Aug 2018
```
Comment (gokulcek) [inline README.md]: can you add date and time

Comment (appars) [reply README.md]: Hi, I have made the changes. Can you please review and approve?

**Comment by appars** [comment]
@gokulcek  Impleneted changes. please review

**Review by gokulcek — APPROVED** [review]
looks good to me.

**Pull request merged**